src/rpc/protocol.cpp

```diff
@@ -114,13 +114,3 @@ bool GetAuthCookie(std::string *cookie_out)
         *cookie_out = cookie;
     return true;
 }
-
-void DeleteAuthCookie()
```

Comment:
Huh, it's a mistake that this is unused; the auth cookie is supposed to be removed on shutdown.